frontend/src/components/history-page/sort-button/sort-button.tsx

```diff
@@ -20,17 +22,8 @@ export enum SortModeEnum {
  * @param direction The sorting direction for which to get the icon
  * @return The name of the icon fitting to the given sorting direction
  */
-const getIcon = (direction: SortModeEnum): IconName => {
-  switch (direction) {
-    case SortModeEnum.no:
-      return 'sort'
-    case SortModeEnum.up:
-      return 'sort-asc'
-    case SortModeEnum.down:
-      return 'sort-desc'
-    default:
-      return 'sort'
-  }
+const getIcon = (direction: SortModeEnum): Icon => {
+  return direction === SortModeEnum.down ? IconSortAlphaDown : IconSortAlphaUp
```

Comment:
SortModeEnum.up is not handled anymore?

Reply:
It is. Not down => up

Reply:
So there is no icon for "unsorted" anymore?

Reply:
It is now. I use the X icon.